apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -82,9 +121,21 @@ export function AiGeneratePage({ resLocals }: { resLocals: Record<string, any> }
         </main>
       </body>
       <script>
+        const questions = {
+          ${examplePrompts
+            .map(
+              (question) =>
+                `${question.id}: {prompt: \`${question.prompt}\`, userPrompt: \`${question.userPrompt}\`, gradingPrompt: \`${question.gradingPrompt}\`}`,
+            )
+            .join(', ')},
```

Comment:
Do not do this. If you must shuttle around the entire data object, use [`EncodedData` and `decodeData`](https://github.com/PrairieLearn/PrairieLearn/tree/master/packages/browser-utils#encodeddata-and-decodedata). That said, I think a much nicer solution would be to add these as a `data-*` attributes on each individual `<option>` above.

Reply:
Changed in 09ebd145316f730bdcbac5525887fda484b63483